common/src/main/java/org/opensearch/ml/common/utils/ModelInterfaceUtils.java

```diff
@@ -558,37 +558,46 @@ private static Map<String, String> createPresetModelInterfaceByConnector(Connect
         if (connector.getParameters() != null) {
             switch ((connector.getParameters().get("service_name") != null) ? connector.getParameters().get("service_name") : "null") {
                 case "bedrock":
-                log.info("Creating preset model interface for Amazon Bedrock model: {}", connector.getParameters().get("model"));
+                log.info("Detected Amazon Bedrock model");
                 switch ((connector.getParameters().get("model") != null) ? connector.getParameters().get("model") : "null") {
                     case "ai21.j2-mid-v1":
+                        log.info("Creating preset model interface for Amazon Bedrock model: {}", connector.getParameters().get("model"));
```

Comment:
Which cases prints these logs, will it overwhelm the log file?

Reply:
Only when model registration and if the automated model interface creation is triggered it will print

Reply:
We can change it to debug level if you want

Reply:
I prefer to log it in debug level.

Reply:
Changed